README.md

```diff
@@ -19,57 +19,72 @@ to provide the cryptographic primitives.
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/rustls-ffi.svg)](https://repology.org/project/rustls-ffi/versions)
 
-# Build
+# Build rustls-ffi
 
-You'll need to [install the Rust toolchain](https://rustup.rs/) (version 1.71
-or above) and a C compiler (`gcc` and `clang` should both work).
+To build rustls-ffi as a static or shared library you'll need to [install the
+Rust toolchain](https://rustup.rs/) (version 1.71 or above) as well as 
+[cargo c](https://github.com/lu-zero/cargo-c).
 
-## Cryptography provider
+The [cargo-c] tool can be installed from 
+[your package manager](https://github.com/lu-zero/cargo-c?tab=readme-ov-file#availability)
+or using Cargo with `cargo install cargo-c`.
 
-Both rustls and rustls-ffi support choosing a cryptography provider for
-implementing the cryptography required for TLS. By default, both will use
-[`aws-lc-rs`][], but [`*ring*`][] is available as an opt-in choice.
+If you plan to build the `client` and `server` [C examples](#example-applications) 
+you will also need `cmake` and a C compiler (`gcc` and `clang` should both work, 
+as well as MSVC on Windows).
 
-It is **not** presently supported to build with both cryptography providers
-activated, or with neither provider activated.
+## Building and Installing
 
-### Choosing a provider
+```bash
+git clone https://github.com/rustls/rustls-ffi
+cd rustls-ffi
+sudo cargo capi install --release
+```
 
-#### Make
+If you receive a message like "error: no such command capi" you need to install
+[cargo c] from your package manager, or using `cargo install cargo-c`.
 
-When building with the `Makefile`, or example `Makefile.pkg-config` specify
-a `CRYPTO_PROVIDER` as a makefile variable. E.g.:
+To change where the library is insatlled, use `--prefix` like:
 
-* `make` to build with the default (`aws-lc-rs`).
-* `make CRYPTO_PROVIDER=aws-lc-rs` to build with `aws-lc-rs` explicitly.
-* `make CRYPTO_PROVIDER=ring` to build with `*ring*`.
+```bash
+cargo capi install --release --prefix=/tmp/rustls-ffi
+```
 
-#### CMake
+Running `capi install` takes care of:
+* Building `.a`, `.so` or `.dylib` library files for rustls-ffi (depending on
```

Comment:
I think `.dll` should be on this list now too, right?

Reply:
Good point. I added `.dll` and `.lib` to cover both dynamic and static linking on Windows.